react/week1/react-props/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import TodoListHeader from './Functional-components/TodoListHeader';
+import TodoItems from './Functional-components/TodoItems';
+import Header from './Class-components/Header';
+import Items from './Class-components/Items';
+
+
+
+const todos = [
+	{
+		id: 1,
+		description: 'Get out of bed',
+		deadlineDate: 'Tue Jan 28 2020',
+	},
+	{
+		id: 2,
+		description: 'Brush Teeth',
+		deadlineDate: 'Wed Jan 29 2020',
+	},
+	{
+		id: 3,
+		description: 'Eat breakfast',
+		deadlineDate: 'Thursday Jan 30 2020',
+	},
+];
+
+// Function App() render - Hooks
```

Comment:
You most likely left this commented code to have it as a hint, and it's totally fine 👍

However, in the real project try to avoid leaving any commented code which you don't need